bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -506,5 +507,347 @@
+				<option value="stop">Stop</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="unit_mode_status">
+		<item-type>String</item-type>
+		<label>Unit Mode Status</label>
+		<description>Unit mode status state</description>
+		<state readOnly="true">
+			<options>
+				<option value="cool">Cool</option>
+				<option value="heat">Heat</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="unit_mode_control">
+		<item-type>String</item-type>
+		<label>Unit Mode</label>
+		<description>Unit mode control</description>
+		<state>
+			<options>
+				<option value="cool">Cool</option>
+				<option value="heat">Heat</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="block_menu_control">
+		<item-type>String</item-type>
+		<label>Block Menu</label>
+		<description>Block menu control</description>
+		<state>
+			<options>
+				<option value="block">Block</option>
+				<option value="no">No</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="space_mode">
+		<item-type>String</item-type>
+		<label>Space Mode</label>
+		<description>Space mode control</description>
+		<state>
+			<options>
+				<option value="comfort">Comfort</option>
+				<option value="eco">Eco</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="eco_mode_target_offset">
+		<item-type>Number:Temperature</item-type>
+		<label>Eco Mode Target Offset</label>
+		<description>Eco mode target offset control</description>
+		<state pattern="%d %unit%"></state>
+	</channel-type>
+
+	<channel-type id="eco_mode_offset">
+		<item-type>Number:Temperature</item-type>
+		<label>Eco Mode Offset</label>
+		<description>Eco mode offset state</description>
+		<state pattern="%d %unit%" readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="comm_alarm_bit_status">
+		<item-type>String</item-type>
+		<label>Communication Alarm Bit Status</label>
+		<description>Communication alarm bit status state</description>
+		<state readOnly="true">
+			<options>
+				<option value="alarm">Alarm</option>
+				<option value="no">No</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="operation">
+		<item-type>String</item-type>
+		<label>Operation State</label>
+		<description>Operation state</description>
+		<state readOnly="true">
+			<options>
+				<option value="alarm">Alarm</option>
+				<option value="cool demand-off">Cool demand off</option>
+				<option value="cool thermo-off">Cool thermo off</option>
+				<option value="cool thermo-on">Cool thermo on</option>
+				<option value="dhw-off">DHW off</option>
+				<option value="heat demand-off">Heat demand off</option>
+				<option value="heat thermo-off">Heat thermo off</option>
+				<option value="heat thermo-on">Heat thermo on</option>
+				<option value="off">Off</option>
+				<option value="swp-off">Swp off</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="outdoor_temp">
```

Comment:
Maybe you can rather use the system channel.

Reply:
Please can you be more specific? I am sorry, but I have no idea what you mean...

Reply:
I mean you could replace
`<channel id="outdoor_temp" typeId="outdoor_temp"></channel>`
By
`<channel id="outdoor_temp" typeId="system.outdoor-temperature"></channel>`
And then remove outdoor_temp

Reply:
thanks, done